packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -14,10 +14,14 @@ interface UseFeedLayout {
   shouldUseFeedLayoutV1: boolean;
 }
 
+export const FeedLayoutV1FeedPages = new Set(
```

Comment:
I think we can maybe modify this hook to either return the raw value as well?
(also open to simply using it directly)

Reply:
Do you mean to have another variable, like `export const FeedLayoutV1Feeds` that would export the array of values that are part of the feed layout v1? If not, please elaborate :)

Reply:
Yeah, but after thinking probably best to just use `useFeature` as we already do this in places.